readthedocs/builds/migrations/0017_builds_deterministic_order_index.py

```diff
@@ -4,14 +4,19 @@
 
 
 class Migration(migrations.Migration):
-    safe = Safe.after_deploy
+    safe = Safe.after_deploy()
     dependencies = [
         ("builds", "0016_add_mkdocs_html_doctype"),
     ]
 
     operations = [
-        migrations.AlterIndexTogether(
-            name="build",
-            index_together={("date", "id"), ("version", "state", "type")},
-        ),
+        # TODO: something changed in Django 5.2 that I need to comment this index together here.
+        # We need to research a little more.
+        #
+        # ValueError: Found wrong number (0) of constraints for builds_build(date, id)
+        #
+        # migrations.AlterIndexTogether(
+        #     name="build",
+        #     index_together={("date", "id"), ("version", "state", "type")},
+        # ),
```

Comment:
I'm not sure if we need to do something else with this, but I don't think so since these are old migrations.